wkhtmltopdf-image/src/main/docker/app.py

```diff
@@ -0,0 +1,123 @@
+#! /usr/bin/env python
+import json
+import os
+import tempfile
+
+from executor import execute
+from werkzeug.exceptions import HTTPException, NotFound
+from werkzeug.routing import Map, Rule
+from werkzeug.wrappers import Request, Response
+from werkzeug.wsgi import wrap_file
+
+"""
+This python server is responsible for generating PDFs from HTML
+using an underlying wkhtmltopdf command-line tool.
+
+To use this application, the user must send a POST request with
+base64 or form encoded encoded HTML content and the wkhtmltopdf Options in
+request data, with keys 'contents' and 'options'.
+The application will return a response with the PDF file.
+"""
+class WkHtmlToPdfServer():
```

Comment:
I re-worked their python server script as a class and added a healthcheck endpoint that just returns 200